lib/ical_filter_proxy/calendar.rb

```diff
@@ -1,24 +1,47 @@
 module IcalFilterProxy
   class Calendar
-    attr_accessor :ical_url, :api_key, :timezone, :filter_rules
+    attr_accessor :ical_url, :api_key, :timezone, :filter_rules, :clear_existing_alarms, :triggers
 
     def initialize(ical_url, api_key, timezone = 'UTC')
       self.ical_url = ical_url
       self.api_key = api_key
       self.timezone = timezone
 
       self.filter_rules = []
+      self.clear_existing_alarms = false
+      self.triggers = []
```

Comment:
Can we call this `alarm_triggers` to be clearer?

Reply:
Done.